Userland/Applications/Spreadsheet/SpreadsheetWidget.cpp

```diff
@@ -429,6 +449,24 @@ void SpreadsheetWidget::redo()
     update();
 }
 
+void SpreadsheetWidget::change_cell_static_color_format(bool background)
```

Comment:
What does "static" mean here?

Reply:
Also, don't use a boolean for these kinds of two-value flags, use a special enum instead. See the style guidelines for more.

Reply:
> What does "static" mean here?

The static means static format is cells that don't change their style depending on the content, like if a value is less than ten turn red. That's why is called `static_color_format`. Should I change it?

Reply:
> Also, don't use a boolean for these kinds of two-value flags, use a special enum instead. See the style guidelines for more.

Something like:
```c++
enum FormatType {
    Background = 0,
    Foreground = 1
}
```

Should I use `VERIFY_NOT_REACHED()` in this case?